ts/IterableInboxMessageListItem.tsx

```diff
@@ -86,18 +86,18 @@ const defaultMessageListLayout = (message: InboxRowViewModel, customizations: Cu
       body,
       createdAt,
       messageRow
-   } = styles
+   } = resolvedStyles
 
-   function messageRowStyle(message: InboxRowViewModel) {
-      return message.last ? {...messageRow, borderBottomWidth: 1} : messageRow 
+   function messageRowStyle(rowViewModel: InboxRowViewModel) {
+      return rowViewModel.last ? {...messageRow, borderBottomWidth: 1} : messageRow 
```

Comment:
As far as I can tell, this is the only use of `last` in the data model, I don't think it's a strong enough use case, especially since the way it's set and maintained isn't intuitive. My suggestion is let `IterableInboxMessageListItem` just take a singular `style` parameter, remove any notion of being a `last` row, and then just apply that style.

Reply:
I can pass down a prop that indicates the message list item is the last item in the list and use this to conditionally style the component. I will make the update.